staging/src/k8s.io/kubectl/pkg/cmd/label/label.go

```diff
@@ -393,6 +393,9 @@ func updateDataChangeMsg(oldObj []byte, newObj []byte) string {
 	msg := "not labeled"
 	if !reflect.DeepEqual(oldObj, newObj) {
 		msg = "labeled"
+		if len(newObj) < len(oldObj) {
+			msg = "unlabeled"
```

Comment:
add the label key?
eg: <labelkey> unlabeled

Reply:
Currently the `kubectl label` response is like `node/10.0.x.x labeled/unlabeled`, it is the same as `node/10.0.x.x tainted/untainted`.
So if want to add the label key, there need more code change.